node/pkg/logscribeconsumer/app.go

```diff
@@ -1,33 +1,53 @@
-package zeropglog
+package logscribeconsumer
 
 import (
 	"context"
 	"encoding/json"
+	"net/http"
 	"os"
 	"strings"
 	"time"
 
-	"bisonai.com/orakl/node/pkg/db"
 	errorsentinel "bisonai.com/orakl/node/pkg/error"
+	"bisonai.com/orakl/node/pkg/utils/request"
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
 )
 
-func New(options ...AppOption) *App {
+func New(options ...AppOption) (*App, error) {
 	consoleWriter := zerolog.ConsoleWriter{Out: os.Stdout}
 
 	c := &AppConfig{
-		StoreInterval: DefaultLogStoreInterval,
-		Buffer:        DefaultBufferSize,
+		StoreInterval:    DefaultLogStoreInterval,
+		Buffer:           DefaultBufferSize,
+		LogscribeEnpoint: DefaultLogscribeEnpoint,
 	}
 	for _, option := range options {
 		option(c)
 	}
-	return &App{
-		StoreInterval: c.StoreInterval,
-		buffer:        make(chan map[string]any, c.Buffer),
-		consoleWriter: consoleWriter,
+
+	resp, err := request.RequestRaw(request.WithEndpoint(c.LogscribeEnpoint))
```

Comment:
```suggestion
	resp, err := request.RequestRaw(request.WithEndpoint(c.LogscribeEndpoint))
```

Reply:
wow, I'm surprised this can happen lol. Thanks for catching 😄

Reply:
ah the typo is everywhere 😆